### PR TITLE
Update security.md

### PR DESCRIPTION
**Pull request by mertenar**
@jaimeo please see email related to this PR, one more change I'd like your help with as well as review of the changes already made

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/service-description/security.md [hunk 1]
```diff
@@ -55,8 +56,7 @@ Identity and access management protects corporate assets and business-critical d
 
 Service | Description
 --- | ---
-Biometric Authentication |	Windows Hello allows users to login using their face or a PIN, making passwords harder to forget or steal. For more information, see [Windows Hello.](https://docs.microsoft.com/windows-hardware/design/device-experiences/windows-hello)
-Multi-factor authentication	| Azure multi-factor authentication more tightly controls access to sensitive functions of the Microsoft Managed Desktop service by providing an additional level of authentication using a mobile phone, as well as self-service password reset. 
+Biometric Authentication |	Windows Hello allows users to login using their face or a PIN, making passwords harder to forget or steal. Customers are responsible for implementing the necessary pre-requisites for their on-premises Active Directory for use of this service in a hybrid configuration. For more information, see [Windows Hello.](https://docs.microsoft.com/windows-hardware/design/device-experiences/windows-hello)
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/security.md:59]:
```suggestion
Biometric Authentication |	Windows Hello allows users to sign in using their face or a PIN, making passwords harder to forget or steal. Customers are responsible for implementing the necessary pre-requisites for their on-premises Active Directory for use of this service in a hybrid configuration. For more information, see [Windows Hello](https://docs.microsoft.com/windows-hardware/design/device-experiences/windows-hello).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor edit

**Comment by mypil** [comment]
@mertenar - please assist to merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. 

Thank you.

**Comment by jaimeo** [comment]
Actually, we say "sign in." Not "log in."

**Pull request merged**